langchain_postgres/v2/async_vectorstore.py

```diff
@@ -303,7 +303,8 @@ async def aadd_embeddings(
             values_stmt = "VALUES (:langchain_id, :content, :embedding"
 
             if not embedding and can_inline_embed:
-                values_stmt = f"VALUES (:langchain_id, :content, {self.embedding_service.embed_query_inline(content)}"  # type: ignore
+                # type: ignore
```

Comment:
Can you move the comment back to the original location in order for the lint to pass?